File: curso-python/ex088.py
from random import randint
from time import sleep

lista = []
jogos = []
print("-="*40)
print("JOGA NA MEGA SENA".center(40))
print("-="*40)
sleep(0.7)
quant = int(input("quantos jogos você quer que eu sorteie: "))

tot = 1

while tot <= quant:
    cont = 0
    while True:
        num = randint(1,60)
        if num not in lista:
            lista.append(num)
            cont += 1
        if cont >= 6:
            break

    lista.sort()
    jogos.append(lista[:])
    lista.clear()
    tot += 1

print(f"-="*3, 'SORTEANDO {quant} jogos','-='*3)

for i, l in enumerate(jogos):
    print(f"jogo {i+1}: {l}")
    sleep(1)
print(f"os números sorteados foram {jogos}")
print("-="*30)

File: javascript-exercicios/ex088.js
const prompt = require("prompt-sync")();

let lista = [];
let jogos = [];

console.log("-=".repeat(40));
console.log("JOGA NA MEGA SENA".padStart(30));
console.log("-=".repeat(40));

let quant = Number(prompt("Quantos jogos você quer que eu sorteie: "));
let tot = 1;

while (tot <= quant) {
    let cont = 0;
    while (true) {
        let num = Math.floor(Math.random() * 60) + 1;
        if (!lista.includes(num)) {
            lista.push(num);
            cont++;
        }
        if (cont >= 6) break;
    }

    lista.sort((a, b) => a - b);
    jogos.push([...lista]);
    lista = [];
    tot++;
}

console.log("-=".repeat(3), `SORTEANDO ${quant} jogos`, "-=".repeat(3));

jogos.forEach((l, i) => {
    console.log(`jogo ${i + 1}: ${l}`);
});

console.log(`Os números sorteados foram: ${JSON.stringify(jogos)}`);
console.log("-=".repeat(30));
